### ISSUE TITLE
Latest Git release does not match Google Play

### ISSUE THREAD

**Issue by TimelessNL** [domoticz/domoticz-android#389]
Hi,

I have a question about the release versions. Why is the latest stable release here at github 0.1.266 while the Play Store has 0.1.336?

Does Google Play feature a beta release instead of a stable?

Regards,
Timeless

**Comment by galadril** [comment]
i forget to create releases... sorry

**Comment by galadril** [comment]
ill try to be active on releases on github!

**Comment by TimelessNL** [comment]
Ah fast reply :).

But future releases are being tagged again? Otherwise there does not seem to be a way of knowing which version it actually is. Only that it is newer.

**Comment by galadril** [comment]
I'll try not to forget ;)
I always update build version.. that one matches the number in the Play Store

**Comment by TimelessNL** [comment]
Ok thanks :)

**Comment by TimelessNL** [comment]
I've been thinking. But why is my ticket closed so fast? This is the second time I've created a ticket here. And they both got closed before the issuer has verified anything.

And as a bonus It has not been solved? Only an excuse why it was not done. It is still possible to create tags. I know that versioning is not fun to do. And that development is the main business. But versioning is atleast as importand.
